template.tpl

```diff
@@ -62,6 +62,20 @@ ___TEMPLATE_PARAMETERS___
     "simpleValueType": true,
     "help": "When this checkbox is enabled the tracker will write verbose logs to the browser's developer console.\nWe suggest turning this on only while testing your tracking script in the GTM preview mode. Disabling it reduces the size of the Leadfeeder tracking script and is a suggested option for the production websites.\n",
     "alwaysInSummary": true
+  },
+  {
+    "type": "TEXT",
+    "name": "tracking_cookie_duration_days",
+    "displayName": "The amount of days until the Leadfeeder tracking cookie expires",
```

Comment:
Perhaps we could mention that it applies only when cookie isn't disabled in LF settings? It might be confusing for companies that have disabled the cookie.

Reply:
Good point. I added this info in both displayName and help.